src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,37 @@
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import utils.Retry;
+import utils.TestDataProviders;
+
+public class CalculatorTest extends BaseTest {
+
+    @Test(dataProvider = "Incoming data for Amount", dataProviderClass = TestDataProviders.class, invocationCount = 3, threadPoolSize = 3)
+    public void amountTest(double number1, double number2) {
+        double amount = calculator.amount(number1, number2);
+        double expectedAmount = number1 + number2;
+        Assert.assertTrue(amount == expectedAmount);
```

Comment:
not a mistake, but better to use:
Assert.assertEquals(expectedAmount, amount);

in other tests as well

Reply:
done